tools/login/actions/login.php

```diff
@@ -113,8 +126,17 @@
 if ($_REQUEST["action"] == "logout") {
     $this->LogoutUser();
     $this->SetMessage(_t('LOGIN_YOU_ARE_NOW_DISCONNECTED'));
-    $this->Redirect(preg_replace('/(&|\\\?)$/m', '', preg_replace('/(&|\\\?)action=logout(&)?/', '$1', $incomingurl)));
-    $this->exit();
+    
+    if (isset($loggedouturl))
```

Comment:
```suggestion
    if (empty($loggedouturl))
```
je suppose que c'était plutôt ce code qui était voulu, non ?

Reply:
Le test check si loggedouturl est défini et fait un traitement en fonction. Ton test fera le traitement si loggedouturl est vide...
Le probleme de empty c'est qu'elle peut renvoyer true or false en fonction du contenu de la chaine. exemple plus haut avec "0". Le mieux serait de combiner les 2 tests if (isset(loggedouturl) && !empty(loggedouturl))